revolv/templates/404.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base/base.html" %}
 {% load staticfiles %}
 
-{% block title %}Page Not Found{% endblock %}
+{% block title %}Page Not Found (404) |{% endblock %}
```

Comment:
It's cause it extends base.html, and I couldn't figure out a better way to get the `Page Not Found (404) | RE-volv` in there. What do you think about that format?